### PR TITLE
DROOLS-3476 Refactor SetProperty/InstanceHeaderCommand(s)

### PR DESCRIPTION
**Pull request by annabaker**
-Created AbstractSetHeaderCommand for common code
-Modified SetProperty/InstanceHeaderCommand to implement new conditionalExecute method
-Added additional tests for null ScenarioGridColumn

@gitgabrio @danielezonca @kkufova

### CODE REVIEW

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/actualcommands/AbstractSetHeaderCommand.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+/**
+ * <b>Abstract</b> <code>Command</code> class to provide common methods used by <code>SetInstanceHeaderCommand</code> and <code>SetPropertyHeaderCommand</code> implementations
+ */
+public abstract class AbstractSetHeaderCommand extends AbstractScenarioSimulationCommand {
+
+    public AbstractSetHeaderCommand() {
+        super(true);
+    }
+
+    protected abstract void conditionalExecute(ScenarioSimulationContext context, ScenarioGridColumn selectedColumn);
```
Comment (danielezonca) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/actualcommands/AbstractSetHeaderCommand.java]: Can you please rename it to `executeIfSelectedColumn`? I think it is more understandable

Comment (annabaker) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/actualcommands/AbstractSetHeaderCommand.java]: Done

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/actualcommands/AbstractSetHeaderCommand.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
```
Comment (kkufova) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/actualcommands/AbstractSetHeaderCommand.java]: > 2019

Comment (annabaker) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/actualcommands/AbstractSetHeaderCommand.java]: fixed

**Comment by annabaker** [comment]
@manstis could you please squash/merge?

**Comment by manstis** [comment]
@annabaker I'd gladly.. please let me know when the build is green.

**Comment by annabaker** [comment]
sorry about that.  good to go

**Pull request merged**